src/tests/functional/plugin/gpu/shared_tests_instances/subgraph_tests/preprocess.cpp

```diff
@@ -29,6 +29,8 @@ inline std::vector<preprocess_func> GPU_smoke_preprocess_functions() {
         preprocess_func(resize_linear_nhwc, "resize_linear_nhwc", 0.01f),
         preprocess_func(resize_cubic, "resize_cubic", 0.01f),
         preprocess_func(resize_dynamic, "resize_dynamic", 0.01f, { ov::Shape {1, 3, 123, 123} }),
+        preprocess_func(crop_basic, "crop_basic", 0.01f),
+        preprocess_func(crop_negative, "crop_negative", 0.01f),
```

Comment:
why threshold is not zero?

Reply:
just followed old style :-) will change if there is no more comments